src/features/steps/SIMCard9151.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import React from 'react';
+
+import { Back } from '../../common/Back';
+import Link from '../../common/Link';
+import Main from '../../common/Main';
+import { Next } from '../../common/Next';
+
+export default () => (
+    <Main>
+        <Main.Content heading="Activate iBasis SIM card">
```

Comment:
Change heading to suggested:
Plug-in SIM card

(as SIM cards are already activated)

Reply:
Good catch. But it sounds like you are asking customers to plug in a new sim card?
I assume you meant it as "additional/add-on".

Maybe greg has a good idea here.

Reply:
`Plug in SIM card`, no hyphen, it's a verb.